### PR TITLE
Allow to start an empty room

### LINKED ISSUE #130: Allow to start an empty group call/room
E.g. when you want to have a call with people outside of your nextcloud, you currently have to call a single user/group before being able to make the call public.

So I guess it should be possible to allow to create empty group calls directly, so you can make that public, without triggering a notification for some users you don't want to even have in your call.

### PR DESCRIPTION
**Pull request by nickvergessen**
@Ivansss 

like the other endpoints `/api/oneToOne` and `/api/group` now there is `/api/new`.

Can you add that to the select two as a custom entry that is always there?

Fix #130

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -108,7 +117,8 @@
 					}
 				});
 			});
-			$('#edit-roomname').on("click", function() {
+
+			$('#edit-roomname').on("click", function(e) {
```
Comment (nickvergessen) [inline js/app.js]: unused variable e, please fix in a new PR

File: js/app.js [hunk 2]
```diff
@@ -119,6 +129,12 @@
 				});
 			});
 
+			$('#edit-roomname').on("select2-selecting", function(e) {
+				if (e.object.type === "createPublicRoom") {
+					OCA.SpreedMe.Rooms.createPublicVideoCall();
+				}
+	        });
```
Comment (nickvergessen) [inline js/app.js]: intendation is off

File: js/app.js (hunk 1 above)
Comment (Ivansss) [inline js/app.js]: Fixed in public-rooms PR.

File: js/app.js (hunk 2 above)
Comment (Ivansss) [inline js/app.js]: Fixed in public-rooms PR.

**Pull request merged**